### PR TITLE
pull videos for generic unfurls CORE-9601

### PR DESCRIPTION
**Pull request by mmaxim**
Pull OpenGraph video for generic unfurls, but don't autoplay them.

### CODE REVIEW

File: go/chat/unfurl/display/display.go [hunk 1]
```diff
@@ -30,55 +43,36 @@ func displayUnfurlGeneric(ctx context.Context, srv types.AttachmentURLSrv, convI
 	res.PublishTime = unfurl.PublishTime
 	res.Description = unfurl.Description
 	if unfurl.Image != nil {
-		res.Image = new(chat1.UnfurlImageDisplay)
-		*res.Image = assetToImageDisplay(ctx, convID, *unfurl.Image, srv)
+		if media, err := assetToImageDisplay(ctx, convID, *unfurl.Image, srv); err == nil {
```
Comment (joshblum) [inline go/chat/unfurl/display/display.go]: log errors?

File: go/chat/unfurl/scrape_generic_scoring.go [hunk 2]
```diff
@@ -98,6 +100,40 @@ func getHrefAndContentAttr(e *colly.HTMLElement) []string {
 	return append(getHrefAttr(e), getContentAttr(e)...)
 }
 
+func getOpenGraphVideo(e *colly.HTMLElement) []string {
+	sibs := e.DOM.Siblings()
```
Comment (joshblum) [inline go/chat/unfurl/scrape_generic_scoring.go]: move below short circuit

File: go/chat/unfurl/scrape_generic_scoring.go [hunk 3]
```diff
@@ -98,6 +100,40 @@ func getHrefAndContentAttr(e *colly.HTMLElement) []string {
...
+	size := sibs.Size()
+	url := e.Attr("content")
+	if len(url) == 0 {
+		return nil
+	}
+	mimeType := "video/mp4"
+	var height, width *int
+	for i := 0; i < size; i++ {
```
Comment (joshblum) [inline go/chat/unfurl/scrape_generic_scoring.go]: why can't you query the siblings directly with a selector?

Comment (mmaxim) [reply go/chat/unfurl/scrape_generic_scoring.go]: I just wanted to get all video information in one place, and not have to have some follow up code to see if I got everything.

File: go/chat/unfurl/display/display.go (hunk 1 above)
Comment (mmaxim) [reply go/chat/unfurl/display/display.go]: They get logged higher up.

**Pull request merged**